### ISSUE TITLE
[14] Developers -> References -> Web not visible

### ISSUE THREAD

**Issue by FrancescoBallerini** [odoo/documentation#4373]
I have recently stumbled in this awsome documentation for Odoo14 web customization

https://www.odoo.com/documentation/14.0/developer/tutorials/web.html

which is currently only reachable by https://www.odoo.com/documentation/14.0/developer/reference/addons/actions.html#client-actions-ir-actions-client 
if you click on "See also: tutorial client actions", but i definitely think this is a MUST for the [odoo14 developer reference](https://www.odoo.com/documentation/14.0/developer.html) .

I see there is a warning at the top of the page, warning people that "tutorial is outdated".

This warning should be there without doubt, considering that new developers which might not be aware of odoo owl transition might randomly land on the page, but I am not sure why this page is hidden at the moment.

Probably some paragraphs are already covered by [cheatsheet](https://www.odoo.com/documentation/14.0/developer/reference/javascript/javascript_cheatsheet.html) and [javascript references](https://www.odoo.com/documentation/14.0/developer/reference/javascript/javascript_reference.html) but there are some core advanced concepts like widget template, qweb rendering, rpc call and so on which are not covered by official documentation if this page is hidden.

It would be cool to have it visible on the main references : )

**Comment by AntoineVDV** [comment]
Hello @FrancescoBallerini, and thanks for the feedback :)

Some parts of that tutorial are indeed not properly covered elsewhere, but we are working on that. Over time, we'll add more and more tutorials and how-to guides covering specific subjects like RPCs, QWeb, etc., starting from branch 14.0.

I don't think that temporarily referencing the "Customizing the web client" tutorial would be a good idea, though. Not only because it doesn't cover OWL (which is problematic starting from branch 16.0) but also because some parts are straight-out outdated (even for 14.0) and misleading.

I'm therefore closing the issue but feel free to reply or re-open it if you think of something else we should consider. ;)

**Comment by FrancescoBallerini** [comment]
Hi @AntoineVDV I understand, thanks for the confirm. 

I was wondering if that was intentional but I was almost sure, in fact I have noticed that recently the javascript documentation has been improved in odoo14 as well, which is really appreciated : ) Basically I still had doubt so I ended up reporting this, it's good to know that odoo14 documentation will be enhanced again  👍

**Comment by AntoineVDV** [comment]
Note that most of the focus is on the Odoo 16 documentation, as we want to cover the "modern" frameworks first.

**Comment by FrancescoBallerini** [comment]
That's understandable, I have looked deeper at the [odoo14 javascript references](https://www.odoo.com/documentation/14.0/developer/reference/javascript/javascript_reference.html), and I realized that rpc call were actually documented there, I'm not sure why but I thought it wasn't. (At first I thought it was added recently but I'm not sure anymore since I checked commit history and did not find a recent commit for it.)

If I can give an input to give more sense to my first comment, I think that [using qweb](https://www.odoo.com/documentation/14.0/developer/tutorials/web.html#using-qweb) and [qweb context](https://www.odoo.com/documentation/14.0/developer/tutorials/web.html#qweb-context) should be reported in js reference somewhere, since for now it is only implicitly shown in an example at the bottom of JS reference page. 

I see that it is documented at [qweb templates](https://www.odoo.com/documentation/14.0/developer/reference/javascript/qweb.html#api), which definitely make sense but since qweb.render is also vastly used in JS framework, from my perspective it could be highlited in JS reference, possibly with an example attached. 

I think that might be the only relevant part that misses from js reference.

Sorry for long talk, I think you have good reasons if you don't report a section or you report it differently than we would expect, so I hope I'm not bothering by sending this additional feedback, thanks again for your time.
